src/commands/artifacts.ts

```diff
@@ -20,4 +23,10 @@ export const builder: CommandBuilder = (yargs: Argv) =>
     })
     .demandCommand()
     .demandOption('rev', 'Please specify the revision')
-    .commandDir('artifacts_cmds');
+    .command(list)
+    .command(download);
```

Comment:
What's the point of this change? Looks like more than just the minor import changes I'm seeing in most other files.

Reply:
It is to overcome `yargs`' dynamic nested command generation. To be able to build with esbuild, all imports should be static whereas `yargs` was doing a directory scan with dynamic imports to build this on-the-fly.